### PR TITLE
Add Enum type to filters

### LINKED ISSUE #146: Add Enum type to filters
Connects to https://github.com/resin-io/resin-ui/issues/1374

### PR DESCRIPTION
**Pull request by LucianBuzzo**
Fixes #146

change-type: patch

### CODE REVIEW

File: src/components/Filters/index.js [hunk 1]
```diff
@@ -56,14 +56,18 @@ const SimpleSearchBox = styled.div`
   }
 `
 
-const FilterWrapper = styled(Box)`
-  position: relative;
-`
+const FilterWrapper = styled(Box)`position: relative;`
```
Comment (thgreasi) [inline src/components/Filters/index.js]: I like it when prettier just hasn't yet decided what to use and randomly changes stuff between versions.
It's like its taste evolves with time or maybe it's a kind of a joke messing with us on reviews.

File: src/components/PineTypes/Enum.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import { h } from 'preact'
+import Select from '../Select'
+import _ from 'lodash'
+
+export const Edit = ({ schema, value, onChange, ...props }) => (
+  <Select {...props} value={value} onChange={e => onChange(e.target.value)}>
+    {_.map(schema.values, item => {
+      if (_.isString(item)) {
+        return <option>{item}</option>
```
Comment (thgreasi) [inline src/components/PineTypes/Enum.js]: I guess that we don't need something like `value={item}`, do we?

File: src/components/PineTypes/Enum.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+export const rules = {
+  is: (target, value) => target === value,
+  'is not': (target, value) => target !== value
+}
+
+export const validate = _.isBoolean
+
+export const Display = ({ data, ...props }) => (
+  <div {...props}>
+    <span>{data ? 'true' : 'false'}</span>
```
Comment (thgreasi) [inline src/components/PineTypes/Enum.js]: Is that correct?

File: src/stories/Filters.js [hunk 4]
```diff
@@ -72,6 +74,8 @@ class FiltersDemo extends Component {
       ],
       schema
     }
+
+    window.comp = this
```
Comment (thgreasi) [inline src/stories/Filters.js]: Is that a debug leftover?

File: src/components/PineTypes/Enum.js (hunk 2 above)
Comment (LucianBuzzo) [reply src/components/PineTypes/Enum.js]: `<option>` tags default to using their inner text as the value. If you need to provide a different label to the value then you can pass in an object here.

File: src/components/PineTypes/Enum.js (hunk 3 above)
Comment (LucianBuzzo) [reply src/components/PineTypes/Enum.js]: Nope, this needs to change :smile:

**Pull request merged**